regress/expected/agtype.out

```diff
@@ -4240,6 +4276,48 @@ SELECT '{"a":null, "b":"qq"}'::agtype ?| '["a","a", "b", "b", "b"]'::agtype;
  t
 (1 row)
 
+SELECT '[1,2,3]'::agtype ?| '[1,2,3,4]';
+ ?column? 
+----------
+ t
+(1 row)
+
+SELECT '{"id": 281474976710658, "label": "", "properties": {"n": 100}}'::agtype ?| '[null,"id"]'::agtype;
+ ?column? 
+----------
+ t
+(1 row)
+
+SELECT '{"id": 281474976710658, "label": "", "properties": {"n": 100}}'::agtype ?| '["id",null]'::agtype;
+ ?column? 
+----------
+ t
+(1 row)
+
+SELECT '{"id": 281474976710658, "label": "", "properties": {"n": 100}}'::agtype ?| '[true,"id"]'::agtype;
```

Comment:
Should true be returned here?
`SELECT '{"id": 281474976710658, "label": "", "properties": {"n": 100}}'::jsonb ?& '{true,"id"}';` returns false.